src/axom/primal/operators/winding_number.hpp

```diff
@@ -197,15 +205,40 @@ int winding_number(const Point<T, 2>& R,
 }
 
 /*!
- * \brief Computes the generalized winding number for a single 2D Bezier curve
+ * \brief Computes the solid angle winding number for a 2D polygon
+ *
+ * Overload function without additional returning parameter
+ */
+template <typename T>
+int winding_number(const Point<T, 2>& R,
+                   const Polygon<T, 2>& P,
+                   bool includeBoundary = false,
+                   double EPS = 1e-8)
+{
+  bool isOnEdge = false;
+  return winding_number(R, P, isOnEdge, includeBoundary, EPS);
+}
+
+/*!
+ * \brief Computes the generalized winding number (GWN) for a single 2D Bezier curve
```

Comment:
[Consistency / Somewhat pedantic, but hopefully not too much]
In some cases, we say that we're computing the GWN for a point w.r.t. a primitive, but this one doesn't mention the point. I think it'd be good to be consistent.

Reply:
Good suggestion! I'm also going through and making much of this page's documentation more consistent with respect to acronyms, particularly when distinguishing (integer) winding numbers from generalized winding numbers, or the GWN.